trin-beacon/src/sync.rs

```diff
@@ -39,15 +39,25 @@ impl BeaconSync {
             .build_with_rpc(portal_rpc)
             .expect("Failed to build portal light client");
 
-        // Run the client
-        loop {
+        // Try sync the client 10 times with 10 seconds interval. Return error if failed after 10
+        // attempts
+        let mut counter = 10;
+        while counter > 0 {
             info!(trusted_block_root = %trusted_block_root, "Starting syncing portal light client ...");
             match client.start().await {
                 Ok(_) => break,
                 Err(e) => {
-                    error!("Error syncing portal light client: {e} Retrying...");
+                    error!("Error syncing portal light client: {e} Retrying in 10 secs...");
```

Comment:
```suggestion
                    error!("Error syncing portal light client attempt {}: {e} Retrying in 10 secs...", 11 - counter);
```

Just a nit, but it would be to see which attempt it failed on, as we already hold the count, and it would be easiest then counting logs.